sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysis.java

```diff
@@ -163,4 +224,30 @@ public static SensitivityAnalysisResult run(Network network,
                                                 ContingenciesProvider contingenciesProvider) {
         return find().run(network, factorsProvider, contingenciesProvider);
     }
+
+    public static SensitivityAnalysisResult run(Network network,
+                                                String workingStateId,
+                                                SensitivityFactorsProvider factorsProvider,
+                                                SensitivityAnalysisParameters parameters,
+                                                ComputationManager computationManager) {
+        return find().run(network, workingStateId, factorsProvider, parameters, computationManager);
+    }
+
+    public static SensitivityAnalysisResult run(Network network,
+                                                String workingStateId,
+                                                SensitivityFactorsProvider factorsProvider,
+                                                SensitivityAnalysisParameters parameters) {
+        return find().run(network, workingStateId, factorsProvider, parameters);
+    }
+
+    public static SensitivityAnalysisResult run(Network network,
+                                                SensitivityFactorsProvider factorsProvider,
+                                                SensitivityAnalysisParameters parameters) {
+        return find().run(network, factorsProvider, parameters);
+    }
+
+    public static SensitivityAnalysisResult run(Network network,
+                                                SensitivityFactorsProvider factorsProvider) {
+        return find().run(network, factorsProvider);
+    }
```

Comment:
As we ask in the review, can you add all missing runAsync method please?

Reply:
I think we get all methods runAsync
We have in the Runner 8 methods runAsync with 8 corresponding methods run
We also hase 8 corresponding methods run in sensitivity Analysis